### PR TITLE
Adding TAC option

### PR DESCRIPTION
**Pull request by get-itips**
closes https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/7843

Customizing video filters is now available to set in the TAC

image

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/meeting-policies-audio-and-video.md [hunk 1]
```diff
@@ -145,16 +145,16 @@ For meetings that need the highest-quality video experience, such as CEO board m
 
 This is a per-user policy. This setting controls whether users can customize their video background in a meeting.
 
-Currently, you can only use PowerShell to set this policy. You can edit an existing Teams meeting policy by using the [Set-CsTeamsMeetingPolicy](/powershell/module/skype/set-csteamsmeetingpolicy) cmdlet. Or, create a new Teams meeting policy by using the [New-CsTeamsMeetingPolicy](/powershell/module/skype/new-csteamsmeetingpolicy) cmdlet, and then assign the policy to users.
+You can use both Teams Admin Center and PowerShell to set this policy. You can edit an existing Teams meeting policy by using the [Set-CsTeamsMeetingPolicy](/powershell/module/skype/set-csteamsmeetingpolicy) cmdlet. Or, create a new Teams meeting policy by using the [New-CsTeamsMeetingPolicy](/powershell/module/skype/new-csteamsmeetingpolicy) cmdlet, and then assign the policy to users.
```
Comment (JohanFreelancer9) [inline Teams/meeting-policies-audio-and-video.md:148]:
```suggestion
You can use both Teams admin center and PowerShell to set this policy. You can edit an existing Teams meeting policy by using the [Set-CsTeamsMeetingPolicy](/powershell/module/skype/set-csteamsmeetingpolicy) cmdlet. Or, create a new Teams meeting policy by using the [New-CsTeamsMeetingPolicy](/powershell/module/skype/new-csteamsmeetingpolicy) cmdlet, and then assign the policy to users.
```

File: Teams/meeting-policies-audio-and-video.md [hunk 2]
```diff
@@ -145,16 +145,16 @@ For meetings that need the highest-quality video experience, such as CEO board m
...
 
-To specify whether users can customize their video background in a meeting, set the **VideoFiltersMode** parameter as follows:
+To specify whether users can customize their video background in a meeting, set the **VideoFiltersMode** parameter (**Select video filters** setting in Teams Admin Center) as follows:
```
Comment (JohanFreelancer9) [inline Teams/meeting-policies-audio-and-video.md:150]:
```suggestion
To specify whether users can customize their video background in a meeting, set the **VideoFiltersMode** parameter (**Select video filters** setting in Teams admin center) as follows:
```

File: Teams/meeting-policies-audio-and-video.md [hunk 3]
```diff
@@ -145,16 +145,16 @@ For meetings that need the highest-quality video experience, such as CEO board m
...
 
-|Setting value in PowerShell |Behavior  |
-|---------|---------|
-|**NoFilters**     |User can't customize their video background.|
-|**BlurOnly**     |User has the option to blur their video background. |
-|**BlurandDefaultBackgrounds**     |User has the option to blur their video background or choose from the default set of images to use as their background. |
-|**AllFilters**     |Use has the option to blur their video background, choose from the default set of images, or upload custom images to use as their background. |
+|Setting value in PowerShell|Setting value in Teams Admin Center |Behavior  |
```
Comment (JohanFreelancer9) [inline Teams/meeting-policies-audio-and-video.md:152]:
```suggestion
|Setting value in PowerShell|Setting value in Teams admin center |Behavior  |
```

File: Teams/meeting-policies-audio-and-video.md [hunk 4]
```diff
@@ -145,16 +145,16 @@ For meetings that need the highest-quality video experience, such as CEO board m
...
+|---------|---------|---------|
+|**NoFilters** |**No filter**    |User can't customize their video background.|
+|**BlurOnly**     |**Background blur only**|User has the option to blur their video background. |
+|**BlurandDefaultBackgrounds**|**Background blur and default images**     |User has the option to blur their video background or choose from the default set of images to use as their background. |
+|**AllFilters**|**All filters**     |Use has the option to blur their video background, choose from the default set of images, or upload custom images to use as their background. |
```
Comment (JohanFreelancer9) [inline Teams/meeting-policies-audio-and-video.md:157]:
```suggestion
|**AllFilters**|**All filters**     |User has the option to blur their video background, choose from the default set of images, or upload custom images to use as their background. |
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by scanum** [comment]
@CarolynRowe PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**